querybook/webapp/hooks/useMounted.ts

```diff
@@ -1,12 +1,15 @@
-import { useEffect, useState } from 'react';
+import { useCallback, useEffect, useRef } from 'react';
 
 export function useMounted() {
```

Comment:
This old `useMounted` hook doesn't work actually.  There is good reading about it https://www.codemzy.com/blog/ismounted-hook-with-useeffect-reactjs